Source/View Controllers/SaveViewController.swift

```diff
@@ -125,7 +128,7 @@ class SaveViewController: NSViewController {
     }
 
     func saveTo(url: URL) {
-        print("Saving to \(url)")
+        logger.info("Saving to \(url)")
```

Comment:
There are a couple more places where the string interpolation might redact info like this URL where the print() statement did not. Please double check to see if they need to have the privacy setting to public.

Reply:
When running within Xcode, all information is logged from OSLog calls.  The `privacy: ` parameter is used when running the app outside of Xcode (as a normal user would).

If you build the app, then launch it from the Finder (it will be in the DerivedData folder) and monitor the logs with Console.app, all of that information would be redacted within Console.app unless marked with the `privacy: public` parameter.

Reply:
Thank you! I now see that. Code has been updated to include `privacy: public` parameter.